ansible/files/mesos-master.conf

```diff
@@ -0,0 +1,10 @@
+description "Mesos master"
+
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+respawn limit 10 5
+
+exec mesos-master --hostname=127.0.0.1 --ip=0.0.0.0 --log_dir=/var/log/mesos --work_dir=/var/lib/mesos --quorum=1 --zk=zk://localhost:2181/mesos
```

Comment:
i would like for this to be consistent and use 127.0.0.1 instead of localhost, or all of them to use localhost.